src/nvim/eval.c

```diff
@@ -700,6 +703,19 @@ void eval_diff(const char *const origfile, const char *const newfile, const char
   set_vim_var_string(VV_FNAME_OUT, NULL, -1);
 }
 
+// Return list of chunks calculated by user.
+list_T* eval_idifffn(const int idx_orig, const int idx_new, char_u *rem_p_dex)
```

Comment:
I think `eval_diff_fn` would be a more suitable name. Maybe also rename `eval_diff` to `eval_diff_expr`?